SW.Serverless.SampleWeb/wwwroot/app.css

```
@@ -0,0 +1,129 @@
+:root{
+  --bg:#eef1f4; --surface:#fff; --surface2:#f5f8f9; --rail:#12202a; --rail-ink:#8fa6b6;
+  --ink:#16202b; --ink2:#4d6072; --dim:#7d91a3; --rule:#dbe3ea;
+  --accent:#0d6a72; --ok:#1f7a52; --ok-bg:#dcefe4; --warn:#8a5a12; --warn-bg:#faf0da;
+  --bad:#9c3328; --bad-bg:#f8e2df; --muted-bg:#e7edf1;
+  --mono:"SFMono-Regular",Menlo,Consolas,monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)